source/_components/hive.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: "Hive"
+description: "Instructions on how to integrate Hive devices with Home Assistant."
+date: 2017-09-24 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hive.png
+ha_category: Hub
+ha_release: 0.62
+ha_iot_class: "Cloud Polling"
+---
+
+
+This Hive component is the main component to setup and integrate all supported Hive devices.  Once configured with the minimum required details it will detect and add all your Hive devices in to Home Assistant, including support for multizone heating.
+
+This component uses the unofficial API used in the official Hive website [https://my.hivehome.com](https://my.hivehome.com) and you will need to use the same Username and Password you use on the Hive website to configure this Hive component in Home Assistant.
```

Comment:
Missing comma: `Hive website [https://my.hivehome.com](https://my.hivehome.com), and you will`

Reply:
Updated